flow/src/main/scala/org/alephium/flow/network/DiscoveryServerState.scala

```diff
@@ -144,6 +145,15 @@ trait DiscoveryServerState {
     pendings --= deadPendings
   }
 
+  def setUnreachable(remote: InetSocketAddress): Unit = {
+    unreachables += remote -> TimeStamp.now().plusMinutesUnsafe(10)
+    remove(remote)
+  }
+
+  def mightReachable(remote: InetSocketAddress): Boolean = {
```

Comment:
naming sounds weird, simply `isReachable` or `mightReach` no?

Reply:
it is might be reachable, but we are not sure in this case. If it's false, then we are sure that it's not reachable. I don't know a better name for this